packages/components/src/input/src/Input.css

```diff
@@ -182,17 +184,22 @@
 .o-ui-number-input:not(.o-ui-input-disabled):focus-within,
 .o-ui-text-area:not(.o-ui-input-disabled).o-ui-input-has-focus textarea,
 .o-ui-input-focus:not(.o-ui-input-disabled):not(.o-ui-has-button-disabled) {
-    box-shadow: var(--o-ui-focus-ring-inset-md) !important;
+    box-shadow: 0 0 0 0.12rem var(--hop-neutral-surface), 0 0 0 0.219rem var(--hop-primary-border-focus) !important;
     outline: transparent;
 }
 
 /* STATES | DISABLED */
 .o-ui-text-input.o-ui-input-disabled,
 .o-ui-number-input.o-ui-input-disabled,
 .o-ui-input textarea:disabled {
-    opacity: var(--o-ui-disabled-opacity);
-    background-color: var(--o-ui-input-background-color);
+    background-color: var(--hop-neutral-surface-disabled);
     cursor: not-allowed;
+    --o-ui-input-border-color: var(--hop-neutral-border-disabled);
+}
+
+/* STATES | ACTIVE */
+.o-ui-text-input.o-ui-input-active {
```

Comment:
When does this gets applied?

Reply:
.

Reply:
The active effect is present when the user enters text inside the field.

<img width="783" alt="Capture d’écran, le 2023-10-16 à 14 30 58" src="https://github.com/gsoft-inc/wl-orbiter/assets/2379339/b7af55fd-04a0-464f-b070-dc886235b26f">